frontend/src/components/pages/CampOverview/MoveCamperModal/PageTwo.tsx

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import {
+  Button,
+  ModalBody,
+  ModalCloseButton,
+  ModalFooter,
+  ModalHeader,
+  Select,
+} from "@chakra-ui/react";
+import { Camper } from "../../../../types/CamperTypes";
+import { CampSession } from "../../../../types/CampsTypes";
+
+type PageTwoProps = {
+  camper: Camper;
+  camperIsSelected: boolean[];
+  campers: Camper[];
+  campSessions: CampSession[];
+  selectedCampSession: string;
+  setSelectedCampSession: (campSession: string) => void;
+  moveCamperModalOnClose: () => void;
+  moveCampers: (camperIds: string[], campSessionId: string) => void;
+};
+
+const PageTwo = ({
+  camper,
+  camperIsSelected,
+  campers,
+  campSessions,
+  selectedCampSession,
+  setSelectedCampSession,
+  moveCamperModalOnClose,
+  moveCampers,
+}: PageTwoProps): JSX.Element => {
+  return (
+    <>
+      <ModalHeader>
+        Move{" "}
+        {camperIsSelected.reduce<number>(
+          (prev, cur) => (cur ? prev + 1 : prev),
+          0,
+        )}{" "}
+        Camper(s)
+      </ModalHeader>
+      <ModalCloseButton />
+      <ModalBody>
+        <p style={{ marginBottom: "20px" }}>
+          You can move the camper(s) to another session in this camp by
+          selecting it below:
+        </p>
+        {campSessions.length > 0 ? (
+          <Select
+            value={selectedCampSession}
+            onChange={(e) => setSelectedCampSession(e.target.value)}
+          >
+            {campSessions.map((campSessionsItem, campSessionsIdx) => (
+              <option
+                key={campSessionsIdx}
+                value={campSessionsItem.id}
+                disabled={
+                  campSessionsItem.id === camper.campSession ||
```

Comment:
I think you missed a check here to disable all sessions which have an end date before today's date. IE: admin shouldn't be able to select a session in the past (session which has completed)